src/kvilib/ext/KviOsInfo.cpp

```diff
@@ -718,6 +718,6 @@ namespace KviOsInfo
 
 	QString qtVersion()
 	{
-		return QString(qVersion());
+		return QString(qtVersion());
```

Comment:
How does this NOT blow up kvirc?
This is infinite recursion which finishes with stack overflow.